ext/rinku/autolink.c

```diff
@@ -313,3 +313,40 @@ sd_autolink__url(
 
 	return link_end;
 }
+
+int
+isUnicodeSpace(uint8_t *data, size_t offset) {
```

Comment:
Reminds me of https://github.com/SamSaffron/fast_blank/blob/master/ext/fast_blank/fast_blank.c